### ISSUE TITLE
Merging images into a gif

### ISSUE THREAD

**Issue by tuanddd** [lukechilds/merge-images#110]
Hi, has anyone bumped in the case where you need to merge multiple images, some of them are gif and would like the output result to be a gif?

Currently I'm getting a static image even though I specified the format to be `image/gif`

Thanks.

**Comment by CharlieAIO** [comment]
yeah im also having this issue, haven't found a solution yet. Please let me know if you did

**Comment by hisRoyalty** [comment]
Same here.

**Comment by cdeck95** [comment]
Would like to do the same here. Following in case anyone finds a way.

**Comment by tuanddd** [comment]
@CharlieAIO @hisRoyalty @cdeck95 I ended up using a separate step to convert those merged images into a gif, you would need to merge each frame and then use a cli tool to combine those frames, output is a gif
